utils/spdx/src/main/kotlin/model/SpdxFile.kt

```diff
@@ -59,7 +59,7 @@ data class SpdxFile(
      * A text relating to a copyright notice, even if not complete. To represent a not present value
      * [SpdxConstants.NONE] or [SpdxConstants.NOASSERTION] must be used.
      */
-    val copyrightText: String,
+    val copyrightText: String = SpdxConstants.NOASSERTION,
```

Comment:
commit-msg: I wonder if the link for [2] is mixed up. Should it be as follows?

[2] https://spdx.github.io/spdx-spec/v2.3/package-information/#717-copyright-text-field

Reply:
Actually, both links should link to the JSON specs, which is clearer in this context. Thanks, fixed.